ui/src/app/shared/requirements/form/requirements.form.component.ts

```diff
@@ -68,6 +66,30 @@ export class RequirementsFormComponent {
         }
     }
 
+    ngOnInit() {
+        this._workerModelService.getWorkerModels().first()
```

Comment:
getWorkerModels().pipe(first(), finalize(...      possible with the current version of rxjs ?